### ISSUE TITLE
Can't append custom content at datepicker

### ISSUE THREAD

**Issue by elalekey** [nathanreyes/v-calendar#549]
Actually i have this code 

`<vc-date-picker
          v-model="selectValue" 
          mode="single"
          color="red"
          :first-day-of-week= "2"
          :class="borderType === 1 ? 'horizontal-select-item-perfect' : borderType === 2 ? 'horizontal-select-item' : '' "
          :attributes="dateAttributes"
          :popover="{ visibility: 'focus' }"
        >
          <input
            slot-scope="{ inputValue  }"
            class="vc-width--colektia"
            readonly
            :value="inputValue"
            placeholder="Sin fecha de entrega"
          />
        </vc-date-picker>`

I'm trying to add a custom content to the datepicker and i'm looking in the documentation but didnt got something helpful, yo necesito replicar algo como esto, donde yo pueda seleccionar las siguientes cosas:

1. Datestart
2. Hour finish
3. Extra config
Arrow. Ready

Basically each of this options will do something.. i just need add it extra content, i don't really care to add a datestart or hour finish, help please!! 

![image](https://user-images.githubusercontent.com/41456459/78462364-3e943780-769f-11ea-92f2-496804a49401.png)

This is my actual datepicker

![image](https://user-images.githubusercontent.com/41456459/78462407-9894fd00-769f-11ea-90b5-abb1c5fd9f4b.png)
